ruby3.2-async.yaml

```diff
@@ -28,13 +28,11 @@ vars:
 pipeline:
   - uses: git-checkout
     with:
-      expected-commit: 9942abde424035f6adeb6b5ad5f00c4b41971efd
+      expected-commit: 367745ddf9494698327971b7f843f01a053a3662
       repository: https://github.com/socketry/async
       tag: v${{package.version}}
 
-  - uses: patch
-    with:
-      patches: 001-remove-signing-key.patch
+  - runs: sed -i '/signing_key/d' ${{vars.gem}}.gemspec
```

Comment:
This line is redundant as it is run by default in the ruby/buid pipeline 

https://github.com/chainguard-dev/melange/blob/fe103190cecea4938e1bbeb88e3179023cc69c2e/pkg/build/pipelines/ruby/build.yaml#L41-#45